test/kcli/plans/bond99.nmconnection

```diff
@@ -0,0 +1,25 @@
+[connection]
+id=bond99
```

Comment:
We had `configure-ovs.sh` bugs with spaces in filenames for bonding slaves.

```
id=System eth1
id=Wired connection 1
id=Wired connection enp6s0
id=Wired connection enp5s0-slave-ovs-clone
```
are standard auto-generated names that cause spaces in the `.nmconnection` files.

See https://bugzilla.redhat.com/show_bug.cgi?id=2101157
and
https://cgit.freedesktop.org/NetworkManager/NetworkManager/tree/src/core/settings/plugins/ifcfg-rh/nms-ifcfg-rh-reader.c#n373

I guess maybe we handle this below with `"cloud-init eth1"`

Reply:
Adding a space to the bond connection name. We can add tests with specific naming later on